.rubocop_todo.yml

```diff
@@ -35,55 +35,3 @@ RSpec/ContextMethod:
 # Configuration parameters: AllowConsecutiveOneLiners.
 RSpec/EmptyLineAfterExample:
   Enabled: false
-
```

Comment:
Did you mean to remove these? I.e. is this why the build is failing, or is this related to the rake task somehow?

Reply:
Yes, when you run rubocop --auto-gen-config it adds any offenses that it finds to that todo file. Once you correct the offenses and rerun auto-gen-config it will no longer show up in there.

Reply:
The reason it is failing is on some rubocop offenses that have nothing to do with any of the files I changed. I think they came in when I merged master, but I'm not quite sure how they could have gotten in master without having them specifically ignored in the rubocop.yml

Reply:
Added the cops to the rubocop.yml and all pass now.